iceoryx_examples/icediscovery/iox_find_service.cpp

```diff
@@ -48,27 +48,38 @@ int main()
         serviceDiscovery.findService(iox::capro::IdString_t{"Radar"},
                                      iox::capro::IdString_t{"FrontLeft"},
                                      iox::capro::IdString_t{"Image"},
-                                     printSearchResult);
+                                     printSearchResult,
+                                     iox::runtime::MessagingPattern::PUB_SUB);
```

Comment:
Can we place the `MessagingPattern` into `popo` or `capro`?
It feels weird to be in `runtime`

Reply:
Placed it in `popo` as the `{Publisher,Subscriber}Options` are in there as well. For me it's not part of the user `ServiceDescription` but part of the [internal representation](https://github.com/eclipse-iceoryx/iceoryx/pull/1115#discussion_r811039750).